src/servicex/did_finder/rucio_adapter.py

```diff
@@ -127,7 +131,7 @@ def list_files_for_did(self, did):
                             'adler32': self.get_adler(f['hash']),
                             'file_size': int(f['size'], 10),
                             'file_events': 0,
-                            'file_path': self.get_paths(f['url'])
+                            'file_path': [self.prefix + elem for elem in self.get_paths(f['url'])]
```

Comment:
Is the list return value something internal to RucioAdaptor? We have an issue to _eventually_ return lists of files, but that should not be part of this PR. Maybe I'm missing something and we only return the first element of the list??

Reply:
the get_paths code is from the metalink support changes, I'm just adding the prefix to all of the uris returned